src/pip/_internal/commands/download.py

```diff
@@ -38,6 +38,7 @@ class DownloadCommand(RequirementCommand):
     def add_options(self) -> None:
         self.cmd_opts.add_option(cmdoptions.constraints())
         self.cmd_opts.add_option(cmdoptions.requirements())
+        self.cmd_opts.add_option(cmdoptions.dependency_groups())
```

Comment:
Should this be in `RequirementCommand`?

Reply:
I didn't include it because I didn't think we'd want the option to be presented for `pip wheel`.

It seems slightly strange to me that the wheel command is a `RequirementCommand`, so it's possible that there's something that I haven't understood. It read to me like `pip wheel` can take requirements for the build environment itself -- did I follow that correctly?

If this is the right call, have I split the option up in a good/sensible way?
If not, I'm happy to expose the option(s, since at least one other is under discussion) on `pip wheel` as well.

Reply:
> I didn't include it because I didn't think we'd want the option to be presented for `pip wheel`.

I think we do want it on `pip wheel` -- the mental model I have is `pip create-wheelhouse` is the longer name for `pip wheel`.

Essentially, it's creating a bundle of wheels that you can pass to pip install with the same requirement strings/arguments. And, `--group` makes sense within that context IMO.